src/cdomains/intDomain.ml

```diff
@@ -609,12 +609,14 @@ struct
 
   let cast_to ?torg ?no_ov t = norm ~cast:true t (* norm does all overflow handling *)
 
+  let interval_threshold_widening_ref = ref @@ get_bool "ana.int.interval_threshold_widening"
```

Comment:
Thanks for the tip, `make test` now passes all tests after adding lazy evaluation of the bool value.

Reply:
We also tried to do it with function that takes unit as mentioned above and also make test passes all tests

Reply:
> `lazy` will misbehave in server mode, because the option will only be read once. Our `ResettableLazy` would work around that, but if we really want to optimize this all the way, mutable variables would have the least overhead. The only thing is, they have to be initialized from options in a function, which is called after the options have been handled and reloaded in the server mode.
> 
> Since there are multiple of these cached options, instead of a bunch of `ref`s, it's more convenient to use a record with `mutable` fields, which also avoids the additional indirection of a `ref`.

Thanks for the feedback! We'll make sure to look into the solution with a record in place of a `ref`. Currently we have a solution with a `ref` variable which passes `make test` and has the best running time out of all attempts. Hence, we now have to optimize this with a record, as you suggested.